src/libs/config.js

```diff
@@ -27,11 +27,6 @@ export const Config = {
 
   getTag: async () => (await getConfig()).tag,
 
-  getFeatureFlag: async (featureName) => {
-    const feature = _.get(await getConfig(), 'features', {});
-    return _.get(feature, featureName, false);
-  },
-
```

Comment:
unused

Reply:
We do still need this, even if we don't have any A/B features to turn on/off. This looks directly at the `features` in the config.json files.